src/autodiscovery/startpoint.go

```diff
@@ -0,0 +1,44 @@
+package autodiscovery
+
+import(
+	"time"
+	"log"
+)
+
+
+func detectStartpoint() (path string) {
+	detected := false
+	mountPoints := make(map[string]bool)
+	points, err := getMountPoints()
+	if err != nil {
+		panic(err)
+	}
+	for _, point := range points {
+		mountPoints[point] = true
+	}
+	log.Print("Please connect you card...")
```

Comment:
what will happen if card is already connected at this moment?

Reply:
Nothing will happen. You'll need to eject it and insert again — then it will be recognized.